### PR TITLE
Update sd handler to return image in request, fix some default values…

### PR DESCRIPTION
**Pull request by siddvenk**
## Description ##
- Updates stable diffusion handler to return content-type image/png to user so client can receive image
- Change some default values from None to empty version of corresponding data type (causes issues in generate if None)
- Add test for stable-diffusion-v1-4 model

### CODE REVIEW

File: engines/python/setup/djl_python/stable-diffusion.py [hunk 1]
```diff
@@ -39,12 +40,10 @@ def __init__(self):
     def initialize(self, properties: dict):
         self.model_dir = properties.get("model_dir")
         self.model_id = properties.get("model_id")
-        self.data_type = properties.get("data_type", "fp32")
+        self.data_type = properties.get("data_type", "fp16")
```
Comment (frankfliu) [inline engines/python/setup/djl_python/stable-diffusion.py:43]: We should make it consistent with other handler:
1. data_type vs dtype, or we support both
2. default value should be None, we cast the data type only if user explicitly set the value

**Review by frankfliu — APPROVED** [review]

File: engines/python/setup/djl_python/stable-diffusion.py (hunk 1 above)
Comment (siddvenk) [reply engines/python/setup/djl_python/stable-diffusion.py:43]: good catch!

File: tests/integration/llm/prepare.py [hunk 2]
```diff
@@ -33,6 +33,11 @@
                 "option.task": "text-generation"}
 }
 
+sd_handler_list = {
+    "stable-diffusion-v1-4": {"option.s3url": "s3://djl-llm/stable-diffusion-v1-4", "option.tensor_parallel_degree": 2,
```
Comment (lanking520) [inline tests/integration/llm/prepare.py:37]:
```suggestion
    "stable-diffusion-v1-4": {"option.s3url": "s3://djl-llm/stable-diffusion-v1-4/", "option.tensor_parallel_degree": 2,
```

File: tests/integration/llm/client.py [hunk 3]
```diff
@@ -114,6 +118,24 @@ def test_ds_raw_model(model):
                 assert float(memory) / 1024.0 < spec["max_memory_per_gpu"]
 
 
+def test_sd_handler(model, model_spec):
+    if model not in model_spec:
+        raise ValueError(f"{model} is not one of the supporting models {list(sd_model_spec.keys())}")
+    spec = sd_model_spec[model]
```
Comment (lanking520) [inline tests/integration/llm/client.py:124]: please also add a worker number check in here

**Review by lanking520 — APPROVED** [review]

File: tests/integration/llm/prepare.py (hunk 2 above)
Comment (siddvenk) [reply tests/integration/llm/prepare.py]: done

File: tests/integration/llm/client.py (hunk 3 above)
Comment (siddvenk) [reply tests/integration/llm/client.py:124]: good catch

**Comment by lanking520** [comment]
Merge it for now, let's fix it in the next PR

**Pull request merged**